### PR TITLE
Feature: Social Login

### PR DESCRIPTION
**Pull request by InnocentVictor**
Here can write some details...

### CODE REVIEW

File: file2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (IdokoInnocent) [inline file2.txt:1]: This line is good

**Review by IdokoInnocent — CHANGES_REQUESTED** [review]
Please make this changes and do anther push

**Comment by InnocentVictor** [comment]
Good

**Review by InnocentVictor — COMMENTED** [review]
Everything look perfect

**Pull request merged**